get_predictions.py

```diff
@@ -57,14 +86,14 @@ def analyze_with_openai(image_path: str, verbose=False):
         "Authorization": f"Bearer {creds.OPENAI_API_KEY}",
     }
     payload = {
-        "model": "gpt-4-vision-preview",
+        "model": "gpt-4o",
```

Comment:
question: Is Omni now the preferred model for vision tasks?

Reply:
yes, vision also started denying requests